### DISCUSSION TITLE
Option to hide (+) sign in the tab bar area

### DISCUSSION THREAD

**Discussion by bio** [wezterm/wezterm#1404] [Ideas]
Currently the user can mask the (+) sign in the tab bar using colors.tab_bar.new_tab settings, but technically the invisible `+` tab element takes some space in the tab bar which can be used by other tabs.

**Comment by phgz** [comment]
Bumping this!

**Comment by gegoune** [comment]
`show_new_tab_button_in_tab_bar = false`

**Comment by bio** [comment]
@gegoune Thanks! It works great in the latest version

https://wezfurlong.org/wezterm/config/lua/config/show_new_tab_button_in_tab_bar.html
Since: 20221119-145034-49b9839f
